libbpf-tools/opensnoop.c

```diff
@@ -229,7 +231,7 @@ int main(int argc, char **argv)
 	struct perf_buffer_opts pb_opts;
 	struct perf_buffer *pb = NULL;
 	struct opensnoop_bpf *obj;
-	__u64 time_end;
+	__u64 time_end = 0;
```

Comment:
This needed to fix compilation warning (`-Wmaybe-uninitialized`).

Reply:
Fixing this as part of clean up in https://github.com/iovisor/bcc/pull/2809

Reply:
Haven't seen this, I'll update PR.